### ISSUE TITLE
Fix test sound cards for mb-audit 1.50

### ISSUE THREAD

**Issue by tomcw** [AppleWin/AppleWin#1274]
#### AppleWin HEAD

mb-audit 1.50: (test "A")
- SDM: NG
```
	Test: 21 0C 00
	Exp:33, Act:FF
	AY reg dump:
	000000..00
	000102..0D
	FFFFFF..FF
	FFFFFF..FF
```
- MEGA Audio: NG
```
	Test: 21 0C 00
	Exp:00, Act:FF
	AY reg dump:
	FFFFFF..FF
	FFFFFF..FF
	FFFFFF..FF
	FFFFFF..FF
```

#### AppleWin 1.30.15

mb-audit 1.42: (test "A")
- SDM: OK
- MEGA Audio: OK (warning: 21 14 00 55 00)

mb-audit 1.50: (test "A")
- SDM: OK (warning: 21 14 00 FF 00)
  - Ctrl+Reset: OK (warning: 13 F1 08 FF 04)
- MEGA Audio: NG
```
	Test: 21 0C 00
	Exp:33, Act:00
	AY reg dump:
	000000..00
	000000..00
	000000..00
	000000..00
```

**Comment by tomcw** [comment]
Fix MEGA Audio: 56cc7793517a746d9e9db51ec0da8172adb34dbe

(Ignore mb-audit version)
![mb-audit_000000000](https://github.com/AppleWin/AppleWin/assets/6696896/7ab0ab4b-c05b-4a4d-a7e2-ef6873d074c6)

Warnings:
1. (21,14,00) TestAYReadHiZ: read 6522.IRA (PortA): expect FF; actual 00
2. (13,F1,08) After CTRL+RESET, read 6522.IRB: expect FF; actual 00

**Comment by tomcw** [comment]
Fix SD Music: 53fddfe41ab0d29d3080811600cd7e5abfb245a3

![mb-audit_000000002](https://github.com/AppleWin/AppleWin/assets/6696896/f4e9aea6-3630-4318-8ab9-b991802e678d)

**Comment by tomcw** [comment]
s4=SD Music and s5=MEGA Audio

![mb-audit_000000003](https://github.com/AppleWin/AppleWin/assets/6696896/e24868bc-6426-48c8-bba9-16d25d335867)
